cypress/support/commands.js

```diff
@@ -342,3 +342,40 @@ Cypress.Commands.add('createResponseTemplate', ({ title, content }) => {
     `utf8=%E2%9C%93&response_template%5Btitle%5D=${encodedTitle}&response_template%5Bcontent%5D=${encodedContent}`,
   );
 });
+
+/**
+ * Enables a feature flag.
+ *
+ * @param {string} flag The name of the feature flag to enable.
+ *
+ * @returns {Cypress.Chainable<Cypress.Response>} A cypress request for enabling a feature flag.
+ */
+Cypress.Commands.add('enableFeatureFlag', (flag) => {
+  return cy.request('POST', '/api/feature_flags', { flag });
+});
+
+/**
+ * Disables a feature flag.
+ *
+ * @param {string} flag The name of the feature flag to disable.
+ *
+ * @returns {Cypress.Chainable<Cypress.Response>} A cypress request for disabling a feature flag.
+ */
+Cypress.Commands.add('disableFeatureFlag', (flag) => {
+  return cy.request('DELETE', `/api/feature_flags?flag=${flag}`);
+});
+
+/**
+ * Checks the current status of a feature flag.
+ *
+ * @param {string} flag The name of the feature flag to check.
+ *
+ * @returns {Cypress.Chainable<Cypress.Response>} A cypress request for checking a feature flag.
+ */
+Cypress.Commands.add('checkFeatureFlag', (flag, expected) => {
+  return cy
+    .request('GET', `/api/feature_flags?flag=${flag}`)
+    .should((response) => {
+      expect(response.body).to.deep.equal({ [flag]: expected });
+    });
+});
```

Comment:
I'm not totally sure this file is the right place for this, as `commands` tends to be for executing actions rather than making assertions.

Maybe for now it should be a helper function in `cypress/integration/seededFlows/toggleFeatureFlags.spec.js` and we can extract to a shared place if we need it elsewhere in future?

Reply:
Yes, fair enough. I did look at the [docs](https://docs.cypress.io/api/cypress-api/custom-commands) and some commands there did include assertions, but never as the main purpose. You're probably right, a helper in the spec itself makes more sense as we're unlikely to need it elsewhere. I'll extract that tomorrow.